magellan-library/src/main/java/com/wealthfront/magellan/navigation/Navigator.kt

```diff
@@ -2,5 +2,7 @@ package com.wealthfront.magellan.navigation
 
 interface Navigator {
 
+  val journey: NavigableCompat
```

Comment:
Is this supposed to be the rootNavigable? Why does every navigator need a reference to a Journey?

Reply:
Each navigator is now associated with its respective journey. This way the navigation delegate has access to the journey and can call the onUpdateMenu in the order that we want.